app/views/team-members/team-members.njk

```diff
@@ -118,6 +118,62 @@ Team members - GOV.UK Pay
       </table>
     </div>
     {% endif %}
+    {% if team_members["view-and-initiate-moto"].length %}
+    <div class="admin-list" id="team-members-view-and-initiate-moto-list">
+      <h3 id="view-and-initiate-moto-role-header" class="govuk-heading-m">
+        View and take telephone payments <span class="govuk-!-font-weight-regular pay-text-grey">({{team_members["view-and-initiate-moto"].length}})</span>
+      </h3>
+      <table class="govuk-table pay-!-border-top">
+        <tbody class="govuk-table__body">
+          {% for team_member in team_members["view-and-initiate-moto"] %}
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell">
+                {% if permissions.users_service_read %}
+                  <a class="govuk-link govuk-link--no-visited-state govuk-!-display-block " id="view-and-initiate-moto-member-{{ loop.index }}-link" href="{{ team_member.link }}">
+                {% else %}
+                  <span>
+                {% endif %}
+                {{ team_member.username }} {% if team_member.is_current %}(you){% endif %}
+                {% if permissions.users_service_read %}
+                  </a>
+                {% else %}
+                  </span>
+                {% endif %}
```

Comment:
Conditional closing tags frightening me a bit! 

Could the condition be lifted for readability?

(suggestion not tested!)

```suggestion
                {% set permissionText %}
                  {{ team_member.username }} {% if team_member.is_current %}(you){% endif %}
                {% endset %}
                {% if permissions.users_service_read %}
                  <a href="{{ team_member.link }}">{{ permissionText }}</a>
                {% else %}
                  <span>{{ permissionText }}</span>
                {% endif %}
```

Reply:
That entire block is just a lightly-modified block copied and pasted from another part of the page but you’re right the logic is hideous so I’ll have a look at fixing it.

Reply:
OK, done that.